hathor/sysctl/feature_activation/manager.py

```diff
@@ -0,0 +1,72 @@
+#  Copyright 2024 Hathor Labs
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from hathor.feature_activation.bit_signaling_service import BitSignalingService
+from hathor.feature_activation.feature import Feature
+from hathor.sysctl.sysctl import Sysctl
+
+
+class FeatureActivationSysctl(Sysctl):
+    def __init__(self, bit_signaling_service: BitSignalingService) -> None:
+        super().__init__()
+        self._bit_signaling_service = bit_signaling_service
+
+        self.register(
+            path='support_features',
+            getter=self.get_support_features,
+            setter=None,
+        )
+        self.register(
+            path='not_support_features',
+            getter=self.get_not_support_features,
+            setter=None,
+        )
+        self.register(
+            path='signaling_features',
+            getter=self.get_signaling_features,
+            setter=None,
+        )
+        self.register(
+            path='add_support',
```

Comment:
You can dynamically create a `support.[feature_name]` so one can assign 0/1 to disable/enable. What do you think?

Reply:
Make sense. Thanks!